src/ios/AppDelegate+notification.m

```diff
@@ -62,7 +62,7 @@ - (void)application:(UIApplication *)application didFailToRegisterForRemoteNotif
     [pushHandler didFailToRegisterForRemoteNotificationsWithError:error];
 }
 
-- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo {
+- (void)applicationPushPlugin:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo {
```

Comment:
Before PushPlugin would implement AppDelegate's didReceiveRemoteNotification.  We do not want PushPlugin to implement didReceiveRemoteNotification.  Instead, we want to implement didReceiveRemoteNotification then call PushPlugin's code when we want to.  So our AppDelegate looks like this:

```
- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo {
    if (/*usePushPlugin*/) {
        [self applicationPushPlugin:application didReceiveRemoteNotification: userInfo];
    } else {
        //Handle remote notification another way
    }
}
```